packages/entryPoints/index.ts

```diff
@@ -306,4 +311,12 @@ export async function startPreview(unityInterface: IUnityInterface) {
       handleServerMessage(message)
     }
   })
+
+  window.addEventListener('message', async (msg) => {
+    if (typeof msg.data === 'string') {
+      if (msg.data.startsWith('{')) {
```

Comment:
whats this magic ? We are sending the code here and also via the gloabLThis.Playground ?
Maybe this could be something more specific ? `msg.data === 'reload-playground'` ?

Reply:
A residual behviour, I'll modify it (it had the CLI messages behaviour replicated)